README.md

```diff
@@ -1,42 +1,34 @@
 # Stacks
 
-This respository holds the default set of stacks avalible from the appsody cli. Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it. A stack is make up of a stack and one or more templates.
-
-**Stack:** A stack is the foundation for your application. A stack can set default functionality and controls what can be overwritten by a template or application based on it.
-
-**Template:** A template is based upon a stack providing a template application that is ready to use. It has access to any dependencies supplied by that stack and is able to include extra dependencies plus new functionality to enhance the stack where allowed.
-
-## Getting started with Stacks
-1. Create app directory
-``` bash
-mkdir my-app && cd my-app
-```
-2. Initalise Appsody stack with template
-``` bash
-appsody init <stack>/<template>
-```
-3. Launch application in development environment
-``` bash
-appsody run
-```
+This respository holds the default set of stacks avalible from the appsody cli. Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it. 
 
```

Comment:
This respository holds the default set of stacks available from Appsody. Each stack is pre-configured with popular open source technologies to enable rapid development of quality microservice-based applications. Stacks include a base container image and project templates which act as a starting point for your application development. 

Stacks allow for rapid development whilst giving the stack developer the ability to control the overall applications that are created from it.